locale/templates/main.ftl

```diff
@@ -344,6 +344,17 @@ plan-details-header = Product details
 plan-details-show-button = Show details
 plan-details-hide-button = Hide details
 plan-details-total-label = Total
+plan-details-list-price = List Price
+
+## coupons
```

Comment:
If this is a section comment, it needs to have an extra line break between this and the strings below. 

See documentation on this; https://projectfluent.org/fluent/guide/comments.html

Reply:
@IvoJP

Reply:
I'm not sure how to handle this request - there isnt an extra line for groups anywhere else in that file so adding it for this section would be inconsistent. I also couldn't find anything in the documentation that recommended/required it (i could be wrong) aside from the example employing the change requested here.